techniques/html/H100.html

```diff
@@ -107,15 +107,15 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
     
     
     <section id="description">
-      <h2>Description</h2>
-      <p>The objective of this technique is to provide examples of properly marked up email and password inputs. This technique involves providing form mechanism where the user can enter their email address and password to log into the website.</p>
-      <p>Browsers and password managers generally use the accessible name of the input to determine how to fill it in. The user agent (including password managers) should not be blocked from populating the fields, and pasting by the user should also not be blocked.</p>
+      <h2>解説</h2>
+      <p>このテクニックの目的は、適切にマークアップされた電子メール及びパスワードの入力の例を提供することである。このテクニックでは、利用者がメールアドレス及びパスワードを記入してウェブサイトにログインできるフォームメカニズムを提供する。</p>
+      <p>ブラウザ及びパスワードマネージャーは通常、入力のアクセシブルな名前 (name) を使用して記入方法を決定する。ユーザエージェント (パスワードマネージャーを含む) によるフィールドへの入力をブロックすべきではなく、利用者による貼り付けもブロックすべきではない。</p>
```

Comment:
fill in ですが、解説書 3.3.9 の方では単に「入力」と訳しているようです。

https://waic.jp/translations/WCAG22/Understanding/accessible-authentication-enhanced
>利用者のブラウザ又は統合されたサードパーティのパスワードマネージャー拡張機能は、入力の目的を特定し、ユーザ名及びパスワードを**自動的に入力**できる。

https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-enhanced.html
>The user's browser or integrated third-party password manager extension can identify the purpose of the inputs and **automatically fill in** the username and password.

これに揃えるなら:

```suggestion
      <p>ブラウザ及びパスワードマネージャーは通常、入力のアクセシブルな名前 (name) を使用して入力方法を決定する。ユーザエージェント (パスワードマネージャーを含む) によるフィールドへの入力をブロックすべきではなく、利用者による貼り付けもブロックすべきではない。</p>
```